docs/UserGuide.md

```diff
@@ -546,7 +545,8 @@ Action | Format, Examples
 **Delete Meeting** | `meeting-delete INDEX i/MEETING-INDEX`<br> e.g., `meeting-delete 1 i/1 `
 **Set as Favourite** | `fav INDEX`<br> e.g., `fav 3`
 **Remove as Favourite** | `unfav INDEX`<br> e.g., `unfav 3`
-**Refresh** | `refresh` <br> e.g., `refresh`
+**Refresh** | `refresh`
```

Comment:
You might want to group this together with the single input commands below instead of being placed between commands that require multiple arguments

Reply:
Reordered